src/main/java/com/ericsson/bss/cassandra/ecaudit/AuditEntryBuilderFactory.java

```diff
@@ -78,7 +79,7 @@ public Builder createEntryBuilder(String operation, ClientState state)
             CQLStatement statement = QueryProcessor.getStatement(operation, state).statement;
             return createEntryBuilder(statement);
         }
-        catch (KeyspaceNotDefinedException e)
+        catch (InvalidRequestException e)
```

Comment:
If we throw an exception from within the catch clause here we would throw it through the stack. Should we have "catch (RunTimeException)" outside of this try/catch?

Reply:
Done